File: app/Models/Tipopago.php
<?php

// app/Models/Tipopago.php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Tipopago extends Model
{ /**
 * App\Models\Tipopago
 *
 * @property int $id
 * @property string $nombre
 * @property \Illuminate\Support\Carbon|null $created_at
 * @property \Illuminate\Support\Carbon|null $updated_at
 */

    use HasFactory;

    protected $table = 'tipospago'; // nombre exacto de la tabla

    protected $fillable = ['nombre'];
}

File: TRIMESTRE 8/Soxmovil-Frontend/Appmovil-android/AppBase/app/src/main/java/com/example/appinterface/Modelos/TipoPago/TipoPago.kt
package com.example.appinterface.Modelos.TipoPago

import java.io.Serializable

data class TipoPago(
    val id: Int? = null,
    val nombre: String,
    val createdAt: String? = null,
    val updatedAt: String? = null
): Serializable